Danki2/Assets/Scripts/Actor/Mortal/Player/Player.cs

```diff
@@ -27,12 +27,17 @@ public enum ActionControlState
 
 public class Player : Mortal
 {
+    public float maximumCooldown = 1f;
+    public float totalDashCooldown = 1f;
+    public float dashDuration = 0.2f;
+    public float dashSpeedMultiplier = 3f;
```

Comment:
I think in general it'd be good to do the editor scripts as part of the same task as implementing features, otherwise I think it's pretty unlikely that we'll come back later to sort them out and it's another task to remember to plan. What do you think?

Reply:
Yeah I think you're right, I was a bit hesitant because Andy is about to componentise player controls - but it's better for him to have something to work from anyway